### ISSUE TITLE
Possible to use other plugins? (imagemin-guetzli etc.)

### ISSUE THREAD

**Issue by lostpebble** [tcoopman/image-webpack-loader#85]
I'd like to make use of the new JPEG compression from Google, https://github.com/google/guetzli/

I see there is an imagemin plugin for it: https://github.com/imagemin/imagemin-guetzli

I've downloaded it and tried to use it by just adding some options to it in the query:

```
{
        test: /\.(jpe?g|png|gif|svg)$/i,
        loaders: ['file-loader?context=src/images&name=images/[path][name].[ext]', {
          loader: 'image-webpack-loader',
          query: {
            guetzli: {
              quality: 95,
            },
            gifsicle: {
              interlaced: false,
            },
            optipng: {
              optimizationLevel: 4,
            },
            pngquant: {
              quality: '75-90',
              speed: 3,
            },
          },
        }],
        exclude: /node_modules/,
        include: __dirname,
      },
```

But this doesn't seem to work. I think it would be cool if we could easily add plugins like this. Or perhaps I'm missing the right way to do it?

**Comment by Milewski** [comment]
Yeah it would be quite easy to add a new plugin into this lib. but probably would be more ideal if we could pass our own optimizers manually

**Comment by Milewski** [comment]
Well i just added it myself with this pull request https://github.com/tcoopman/image-webpack-loader/pull/86

**Comment by ShawnCorrigan** [comment]
Has this been resolved? Not specific to imagemin-guetzli, are we able to somehow pass in other imagemin plugins?

**Comment by milewski** [comment]
@ShawnCorrigan https://github.com/milewski/imagemin-loader i made a version that u can pass any plugin manually

**Comment by ShawnCorrigan** [comment]
@milewski Ah thanks, I made a PR with the ability to pass any plugin, but I guess it's not likely to be accepted.

**Comment by milewski** [comment]
i did that too.. https://github.com/tcoopman/image-webpack-loader/pull/86 but never got a response
